src/components/loader-spinner/loader-spinner.pw.tsx

```diff
@@ -0,0 +1,223 @@
+import React from "react";
+import { test, expect } from "@playwright/experimental-ct-react17";
+import {
+  loaderSpinnerWrapper,
+  loaderSpinnerCircleSvg,
+  loaderSpinnerVisibleLabel,
+  loaderSpinnerHiddenLabel,
+} from "../../../playwright/components/loader-spinner/index";
+import {
+  checkAccessibility,
+  getStyle,
+} from "../../../playwright/support/helper";
+import LoaderSpinnerComponent from "./components.test-pw";
+import { LOADER_SPINNER_SIZE_PARAMS } from "./loader-spinner.config";
+import { LoaderSpinnerColors } from "./loader-spinner.component";
+
+const sizes = ["XS", "S", "M", "L", "XL"] as const;
+
+test.describe("Prop checks for Loader Spinner component", () => {
+  sizes.forEach((spinnerSizes) => {
+    test(`when the 'size' prop is passed as '${spinnerSizes}' the wrapper has the correct dimensions`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<LoaderSpinnerComponent size={spinnerSizes} />);
+
+      await expect(loaderSpinnerWrapper(page)).toHaveCSS(
+        "height",
+        `${LOADER_SPINNER_SIZE_PARAMS[spinnerSizes].wrapperDimensions}px`
+      );
+      await expect(loaderSpinnerWrapper(page)).toHaveCSS(
+        "width",
+        `${LOADER_SPINNER_SIZE_PARAMS[spinnerSizes].wrapperDimensions}px`
+      );
+    });
+  });
+
+  sizes.forEach((spinnerSizes) => {
+    test(`when the 'size' prop is passed as '${spinnerSizes}' the wrapper has the correct box shadow`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<LoaderSpinnerComponent size={spinnerSizes} />);
+
+      const wrapperShadow = await getStyle(
+        loaderSpinnerWrapper(page),
+        "box-shadow"
+      );
+
+      expect(wrapperShadow).toContain(
+        `${LOADER_SPINNER_SIZE_PARAMS[spinnerSizes].boxShadow}px`
+      );
```

Comment:
The `box-shadow` css value value includes more than just this, `toHaveCss` would fail as what we're testing against is not an exact match.

That's why I've went for the `toContain` approach here 👍